### ISSUE TITLE
"You don't have enough privileges to run this operation . . ."

### ISSUE THREAD

**Issue by emirotin** [balena-io/balena-cli#472]
The user on OSX 10.10 has this error when running device init. It happens after the password warning (but the prompt is not shown so the user is probably having passwordless sudo) so it's related to the os init command that involves writing the image.

```
You don't have enough privileges to run this operation.
Try running this command again prefixing it with sudo.

If this is not the case, and you're trying to burn an SDCard, check that the write lock is not set.

If you need help, don't hesitate in contacting us at:

GitHub: https://github.com/resin-io/resin-cli/issues/new
Forums: https://forums.resin.io

You can safely remove /dev/disk2 now
Done
```

This means EPERM / EACCESS error is thrown from the resin internal osinit command

<img src="https://frontapp.com/assets/img/icons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_1hgn)

**Comment by emirotin** [comment]
As the very minimum if the spawned `sudo` command fails we should crash the main command, seems not to be the case right now.

We should also pass down the DEBUG flag to the spawned commands (how do we do it cross-platform?)

**Comment by pdcastro** [comment]
Related to #385

**Comment by pdcastro** [comment]
Closed in favour of #1722.
